app/l10n/locales/az/messages.json

```diff
@@ -483,7 +483,7 @@
   "cQxjXv": "Qalereya",
   "vPemUP": "Kateqoriya",
   "G6669w": "{0}",
-  "FIkhWn": "HAQQIMIZDA",
+  "FIkhWn": "KB",
```

Comment:
Does that mean Kilobyte? This looks like it should be pluralform in the original.

Reply:
The source string is "KB"